src/Carousel.js

```diff
@@ -134,6 +134,8 @@ class Carousel extends Component {
         const { current, dragging } = this.state;
         const offset = this.offset();
 
+        const nonNullChildren = React.Children.toArray(children).filter((c) => c);
```

Comment:
When I tested `React.Children.toArray(children)` seemed to do the job of removing null children. 

Will the filter help with other potential cases?

Reply:
I was unable to use just `toArray`, still had the issue of trying to access `props` of `undefined`. Let me check if I did something wrong.

Reply:
Correct, `filter` is not needed here. This is an undocumented side effect of `toArray` method, though. I guess it makes sense if you consider how React works.